server/index.js

```diff
@@ -13,25 +13,29 @@ app.use(express.static(path.join(__dirname, '../client/dist')));
 app.use('/funding-rounds', (req, res) => {
   fetch(`http://localhost:3001${req.originalUrl}`)
     .then(fetchRes => fetchRes.json())
-    .then(data => res.send(data));
+    .then(data => res.send(data))
+    .catch(() => res.sendStatus(404));
```

Comment:
Honestly, everything looks good. I know sending status is absolutely sufficient. Would it be at all a good idea to also log the error, as shown here? https://expressjs.com/en/guide/error-handling.html

Reply:
Hi Jessica, thanks for good suggestion! I originally removed them because of linter throwing a fit, but discovered that you can turn off a rule per file! Update has been made.